Gems/Atom/RHI/Vulkan/Code/Source/RHI/DescriptorSetLayout.cpp

```diff
@@ -159,6 +159,31 @@ namespace AZ
             const AZStd::span<const RHI::ShaderInputSamplerDescriptor> samplerDescs = m_shaderResourceGroupLayout->GetShaderInputListForSamplers();
             const AZStd::span<const RHI::ShaderInputStaticSamplerDescriptor>& staticSamplerDescs = m_shaderResourceGroupLayout->GetStaticSamplers();
 
+            // About VK_SHADER_STAGE_ALL...
+            // We attempted to configure the descriptor set with the actual resource visibility but it was problematic...
```

Comment:
Suggest TLDR due to length - perhaps single sentence of the problem and current solution, and then dive into the long description.